### PR TITLE
broaden scope

### PR DESCRIPTION
**Pull request by samuelweiler**

### CODE REVIEW

File: charter-draft.html [hunk 1]
```diff
@@ -163,20 +181,27 @@ <h2 id="participation">
...
 			</div>
 			<div class="communication">
 				<h2 id="communication">
 					Communication
 				</h2>
 				<p>
-					This group conducts its work on the public mailing list <strong><a href="https://lists.w3.org/Archives/Public/public-privacy/">public-privacy</a></strong> and in periodic phone calls, typically 1-2 times per month.
+					This group conducts its work on the public mailing list <strong><a href="https://lists.w3.org/Archives/Public/public-privacy/">public-privacy</a></strong>
+					and in periodic teleconferences, typically 1-2 times per month.  Additionally, the group meets
+					face to face at TPAC.  PING is experimenting with the use of Slack
```
Comment (tjwhalen) [inline charter-draft.html]: PING is experimenting with using a free Slack workspace.

File: charter-draft.html [hunk 2]
```diff
@@ -103,33 +107,47 @@ <h1 id="title">
...
+						in their specifications.</li>
 					<li>	Systematizing the privacy review of Web standards.</li>
 				</ul>
 			</div>
 			<div>
 				<h2 id="deliverables">
 					Deliverables
 				</h2>
-				<p>	PING may publish documents consistent with the above scope such as
+				<p>	In conjunction with W3C's <a href="https://www.w3.org/tag">Technical Architecture Group (TAG)</a>
```
Comment (tjwhalen) [inline charter-draft.html]: (A comment, not editorial...)
I'm hoping we can expand this further as we start new initiatives--like the "big ideas"--but for now, these are the concrete docs we can point to, which help.

File: charter-draft.html [hunk 3]
```diff
@@ -47,7 +47,11 @@ <h1 id="title">
 				DRAFT Privacy Interest Group Charter
```
Comment (tjwhalen) [inline charter-draft.html]: I think the charter is much improved!

File: charter-draft.html [hunk 4]
```diff
@@ -47,7 +47,11 @@ <h1 id="title">
...
 			</h1>			
 			<p class="mission">
-				The <strong>mission</strong> of the <a href="https://www.w3.org/Privacy/">Privacy Interest Group</a> is to improve the support of privacy in Web standards by monitoring ongoing privacy issues that affect the Web, investigating potential areas for new privacy work, and providing guidelines and advice for addressing privacy in standards development.
+				The <strong>mission</strong> of W3C's <a href="https://www.w3.org/Privacy/">Privacy Interest Group</a>
+				(PING) is to improve privacy on the Web by advising groups
+				developing standards on how to avoid and mitigate privacy issues with their
+				technologies and by incubating new work that reduces fingerprinting surface area
```
Comment (tjwhalen) [inline charter-draft.html]: debating whether fingerprinting is overly specific for "mission"...not going to fight it, as it's something that comes up a lot, but it does make it seem very central if it's in the mission!

File: charter-draft.html (hunk 1 above)
Comment (samuelweiler) [reply charter-draft.html]: finished that sentence and added some words to reassure people about how we're using slack

**Pull request merged**